src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -30,6 +30,7 @@ use crate::utils::render_tests::{add_flags_and_try_run_tests, try_run_tests};
 use crate::{CLang, DocTests, GitRepo, Mode, PathSet, envify};
 
 const ADB_TEST_DIR: &str = "/data/local/tmp/work";
+const RUSTDOC_JS: &str = "rustdoc-js";
```

Comment:
Suggestion: for *this* PR, could you just inline the constant as strings? String checking is not ideal, but let's keep it consistent with the other test modes/suites for the time being.

Reply:
yeah alright, will do.